inc/switchtec/mfg.h

```diff
@@ -105,6 +119,9 @@ struct switchtec_security_cfg_state {
 
 	bool otp_valid;
 	struct switchtec_security_cfg_otp_region otp;
+
+	bool attestation_supported;
```

Comment:
Why not just have another value in attestation_mode indicating the hardware doesn't support it? Instead of another bool.